Apromore-Frontend/src/processmap/bpmnModelWrapper.js

```diff
@@ -36,9 +36,17 @@ export default class BPMNModelWrapper {
         this._editorApp = new Apromore.BPMNEditor.EditorApp({
             id: editorContainerId,
             fullscreen: true,
-            enabledPlugins: [window.Apromore.I18N.View.zoomIn,
-                            window.Apromore.I18N.View.zoomOut,
-                            window.Apromore.I18N.View.zoomFitToModel]
+            disabledButtons: [

```

Comment:
Just wondering if it would be better to keep a list of enabledButtons instead of disabledButtons. If we were to add a new plugin in the future, we would have to add it to the disabledButtons list of each editor app we don't want it in. (e.g. loganimation)